cloudflare_worker/worker.js

```diff
@@ -62,9 +66,13 @@ function rolloutGroupHeaders(request){
   let isNew = false  // is the group newly-assigned?
 
   // Determine which group this request is in.
-  let controlAssignment = presortToControl(request, control_cookie_pattern, control_group_name)
+  let controlAssignment = presortToControl(request, control_cookie_pattern, control_group_name, killswitch)
```

Comment:
remove this param, then :shipit:

Reply:
Added it back in, currently was setting the cookie a whole bunch of times when we don't need to do so if the killswitch is on and they already have a forced cookie.